test/components/border_box_component_test.rb

```diff
@@ -91,4 +91,12 @@ def test_strips_denied_system_arguments
 
     refute_selector(".p-4")
   end
+
+  def test_renders_row_with_scheme
+    render_inline(Primer::BorderBoxComponent.new) do |component|
+      component.row(scheme: :yellow) { "Row row row your boat" }
```

Comment:
could we add tests for all colors? we could use the constant to define tests similar to https://github.com/primer/view_components/blob/22505f6ed32059d9c15e4d8dd557838b89052fea/test/components/truncate_test.rb#L14-L20